ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -45,6 +45,9 @@ bash -c ' \
     else \
       curl -s -L https://download.ceph.com/ceph-iscsi/2/rpm/el__ENV_[BASEOS_TAG]__/ceph-iscsi.repo -o /etc/yum.repos.d/ceph-iscsi.repo ; \
     fi ; \
+  fi ; \
+  if [ -n "__VAULT_PACKAGES__" ] && [[ $(arch) == "x86_64" ]]; then \
```

Comment:
arm64 rpm package isn't available but the precompiled binary is available for that architecture [1]

Is it worth to have both architectures supported for this ? If yes, we need to change the install method.

[1] https://releases.hashicorp.com/vault/1.7.2/vault_1.7.2_linux_arm64.zip

Reply:
The good and bad thing with the repo is that we don't have to worry about the new version and security fixes. The downside is obviously that the upgrade could break something.
I've been thinking of using the binary directly to accommodate arm64 but realized it was not heavily used as a feature (yet) and the arm64 user base is small.

Honestly, I just feel lazy to go and upgrade the vault version from time to time... :)